jbpm-runtime-manager/src/main/java/org/jbpm/runtime/manager/impl/tx/DisposeSessionTransactionSynchronization.java

```diff
@@ -42,7 +45,7 @@ public void afterCompletion(int status) {
 	    try {
 	        manager.disposeRuntimeEngine(runtime);
 	    } catch (Throwable e) {
-	        // catch exception as it's only clean up and should not affect runtime
+	        logger.warn("Could not dispose runtime engine", e);
```

Comment:
in normal cases this should not fail unless invoked multiple times which then could lead into false alarms - like mixing manual dispose with automatic dispose done on transaction completion